pallets/dapp-staking-v3/src/lib.rs

```diff
@@ -386,6 +386,8 @@ pub mod pallet {
         NoExpiredEntries,
         /// Force call is not allowed in production.
         ForceNotAllowed,
+        /// Account doesn't have the freeze inconsistency
+        InvalidAccount, // TODO: can be removed after call `fix_account` is removed
```

Comment:
nit: can be better named, `AccountNotInconsistent` ?

Reply:
I don't think there's the need for this. If account is inconsistent then there's noop tx paying just fee

Reply:
@ashutoshvarma sure. Believe it or not, that was the first name I choose, but decided to go with a more generic one later xD.

@ermalkaleci 
I added it to make testing more clear.
It will be removed later anyways.

Reply:
fair enough